docs/future-work/voting-improvements/generalize-voting-to-associative-connections.md

```diff
@@ -1,4 +1,8 @@
 ---
 title: Generalize Voting to Associative Connections
 ---
-Needed when each LM learns different representations of the same object (object_id in the code) which is the case in the brain.
\ No newline at end of file
+Currently, voting relies on all learning modules sharing the same object ID for any given object, as a form of supervised learning signal. Thanks to this, they can vote on this particular ID when communicating with one-another.
+
+However, in the setting of unsupervised learning, the object ID that is associated with any given model is unique to the parent LM. As such, we need to organically learn the mapping between the object IDs that occur together across different LMs, such that voting can function without any supervised learning signal.
```

Comment:
You could also mention that the brain has to do the same thing. It could not use a globally consistent SDR representation for each object. The neurons just do associative learning and a cortical column has no idea what the incoming spikes mean.